tests/api/task1.test.js

```diff
@@ -0,0 +1,43 @@
+import {test, expect} from '@playwright/test';
+
+test.describe("API testing", () => {
+
+  const baseUrl = "http://localhost:3030/"
+
+  test("Get all Lists with their List Items", async ({request}) => {
+
+    const response = await request.get(`${baseUrl}/lists`)
+    expect(response.status()).toBe(200)
+
+    const responseBody = JSON.parse(await response.text())
+    console.log(responseBody)
```

Comment:
Let's change it to something more meaningful. For example, expect responseBody to not be empty.